es-core/src/components/TextComponent.cpp

```diff
@@ -120,6 +120,9 @@ void TextComponent::render(const Transform4x4f& parentTrans)
 			case ALIGN_CENTER:
 				yOff = (getSize().y() - textSize.y()) / 2.0f;
 				break;
+			default:
```

Comment:
Added default case to remove compiler warning:

/home/devel/temp/EmulationStation_Stable/es-core/src/components/TextComponent.cpp:112:10: warning: en
umeration values 'ALIGN_LEFT' and 'ALIGN_RIGHT' not handled in switch [-Wswitch]
                switch(mVerticalAlignment)
                       ^~~~~~~~~~~~~~~~~~